api/types.proto

```diff
@@ -747,3 +755,31 @@ message ManagerStatus {
 	// Reachability specifies whether this node is reachable.
 	RaftMemberStatus.Reachability reachability = 4;
 }
+
+// SecretReference is the linkage between a service and a secret that it uses.
+message SecretReference {
+	// Name is the name of the secret that this references.
```

Comment:
Good point.  I've also moved this to the end of the message (although I don't know that ordering implies any sort of importance).